core/sequences/sequences-docs.factor

```diff
@@ -678,7 +678,7 @@ HELP: replace-slice
 { $description "Replaces a range of elements beginning at index " { $snippet "from" } " and ending before index " { $snippet "to" } " with a new sequence." }
 { $errors "Throws an error if " { $snippet "new" } " contains elements whose types are not permissible in " { $snippet "seq" } "." } ;
 
-{ push prefix suffix } related-words
+{ push push-all push-either push-if pop pop* prefix suffix suffix! } related-words
```

Comment:
Personally, I like lots of cross-references, as they help discover related functionality when you know approximately what you need, but not sure of what exactly your options are. But it's up to you if you think I went too far in this case.

As to the specifics, the `prefix` and `suffix` were in one list before I chimed in, so it's not like I smashed together unrelated things that weren't there. I merely expanded the list of synonyms. That's how I see it, anyway.

Good point about the `push-all` and `append`! That was a little thoughtless of me.